nipype/interfaces/freesurfer/preprocess.py

```diff
@@ -2397,23 +2422,44 @@ class ConcatenateLTAOutputSpec(TraitedSpec):
 
 
 class ConcatenateLTA(FSCommand):
-    """concatenates two consecutive LTA transformations
-    into one overall transformation, Out = LTA2*LTA1
+    """ Concatenates two consecutive LTA transformations into one overall
+    transformation
+    
+    Out = LTA2*LTA1
 
     Examples
     --------
     >>> from nipype.interfaces.freesurfer import ConcatenateLTA
     >>> conc_lta = ConcatenateLTA()
-    >>> conc_lta.inputs.in_lta1 = 'trans.mat'
-    >>> conc_lta.inputs.in_lta2 = 'trans.mat'
+    >>> conc_lta.inputs.in_lta1 = 'lta1.lta'
+    >>> conc_lta.inputs.in_lta2 = 'lta2.lta'
+    >>> conc_lta.cmdline # doctest: +ALLOW_UNICODE
+    'mri_concatenate_lta lta1.lta lta2.lta concat.lta'
+
+    You can use 'identity.nofile' as the filename for in_lta2, e.g.:
+
+    >>> conc_lta.inputs.in_lta2 = 'identity.nofile'
+    >>> conc_lta.inputs.invert_1 = True
+    >>> conc_lta.inputs.out_file = 'inv1.lta'
     >>> conc_lta.cmdline # doctest: +ALLOW_UNICODE
-    'mri_concatenate_lta trans.mat trans.mat trans-long.mat'
+    'mri_concatenate_lta -invert1 lta1.lta identity.nofile inv1.lta'
+
+    To create a RAS2RAS transform:
+
+    >>> conc_lta.inputs.out_type = 'RAS2RAS'
+    >>> conc_lta.cmdline # doctest: +ALLOW_UNICODE
+    'mri_concatenate_lta -invert1 -out_type 1 lta1.lta identity.nofile inv1.lta'
     """
 
     _cmd = 'mri_concatenate_lta'
     input_spec = ConcatenateLTAInputSpec
     output_spec = ConcatenateLTAOutputSpec
 
+    def _format_arg(self, name, spec, value):
+        if name == 'out_type':
+            value = {'VOX2VOX': 0, 'RAS2RAS': 1}[value]
+        return super(ConcatenateLTA, self)._format_arg(name, spec, value)
+
     def _list_outputs(self):
```

Comment:
perhaps this can be removed?

Reply:
I'm guessing something else would be needed to change? Removing `_list_outputs` gets me with an `<undefined>` `outputs.out_file`.

Reply:
`name_source` in outputspec

Reply:
I'm not able to figure out how to use `name_source`. Has no effect I can see in output spec, and trying to follow the examples in the [docs](https://nipype.readthedocs.io/en/latest/devel/interface_specs.html) results in failing to generate a string for `inputs.out_file` and the interface fails.

Reply:
nevermind - the logic i had in mind is only implemented if `name_source` is in the input trait. i think the next refactor needs to take care of filename generation